src/Actions/RelativeLineNumbers.ts

```diff
@@ -27,6 +33,12 @@ export class ActionRelativeLineNumbers {
                     return Promise.resolve(false);
                 }
 
+                const lineNumbersTurnedOff = activeTextEditor.options.lineNumbers === TextEditorLineNumbersStyle.Off
+
+                if (lineNumbersTurnedOff) {
+                    return Promise.resolve(false)
```

Comment:
Cloud you please add `;` to line endings so that the coding style is consistent?

Reply:
Of course! How would you feel about adding [Prettier](prettier.io) so that this kind of thing is easier? Happy to do that in a PR :)

Reply:
Great idea! Let's do that!

Reply:
PR incoming :)